doc/api.yml

```diff
@@ -338,12 +338,19 @@ components:
             - ARGENT
             - MATERIEL
             - FLUXARGENT
+            - GROUPEPOSSESSION
+            - ACHATMATERIELAUCOMPTANT
+            - TRANSFERTARGENT
```

Comment:
use `_` to separate words and make them more readable

Reply:
done